homeassistant/components/heatmiser/climate.py

```diff
@@ -5,72 +5,81 @@
 https://home-assistant.io/components/climate.heatmiser/
 """
 import logging
-
+import time
 import voluptuous as vol
 
 from homeassistant.components.climate import ClimateDevice, PLATFORM_SCHEMA
 from homeassistant.components.climate.const import (
-    SUPPORT_TARGET_TEMPERATURE)
+    STATE_AUTO, STATE_COOL, STATE_HEAT, STATE_ECO, SUPPORT_TARGET_TEMPERATURE,SUPPORT_OPERATION_MODE)
 from homeassistant.const import (
-    TEMP_CELSIUS, ATTR_TEMPERATURE, CONF_PORT, CONF_NAME, CONF_ID)
+    TEMP_CELSIUS, ATTR_TEMPERATURE, CONF_PORT, CONF_NAME, CONF_ID, PRECISION_WHOLE, STATE_OFF, STATE_ON, STATE_UNKNOWN)
+    
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['heatmiserV3==0.9.1']
+REQUIREMENTS = ['heatmiserV3==1.1.8']
 
 _LOGGER = logging.getLogger(__name__)
 
 CONF_IPADDRESS = 'ipaddress'
+CONF_PORT = 'port'
```

Comment:
`from homeassistant.const import CONF_PORT`

Reply:
thanks for your help :-) not sure what i need to do here as I'm a python noob! haha I've just been hacking to try fix the original component is it because theirs a conflict in variable names with the home assistant constants variables??

Reply:
`CONF_PORT` is already defined in the main home assistant const file, so we should just import from there instead of redefining it.